src/specBuilder/bar/barUtils.test.ts

```diff
@@ -230,6 +230,27 @@ describe('barUtils', () => {
 			expect(horizontal?.cornerRadiusBottomRight).toEqual(vertical?.cornerRadiusTopRight);
 			expect(horizontal?.cornerRadiusBottomLeft).toEqual(vertical?.cornerRadiusBottomRight);
 		});
+		test('corner radius should be 0 when the hasSquareCorners prop is true', () => {
+			const squareRadius = getCornerRadiusEncodings({ ...defaultBarProps, hasSquareCorners: true });
+
+			// Square radius should have values of 0
+			expect(squareRadius).toEqual(
+				expect.objectContaining({
```

Comment:
I didn't know about `expect.objectContaining`. Very cool. The more you know.

Reply:
It's a good way to only check about the properties you care about, regardless of the rest of the object shape.